packages/main/src/types/ItemSelectionMode.ts

```diff
@@ -0,0 +1,25 @@
+/**
+ * Menu item selection modes.
+ * @public
+ */
+enum ItemSelectionMode {
+	/**
+	 * default type (no item selection)
+	 * @public
+	 */
+	None = "None",
+
+	/**
+	 * Single item selection mode
```

Comment:
The following enum values are preferred as already used in several components with selection mode:
	None = "None",
	Single = "Single",
	Multiple = "Multiple",

Reply:
Personally I have no preference for this, but as we already used `SingleSelect`/`MultiSelect` in the same functionality in OpenUI5, I made it the same for consistency. If you want  - I will change it, or if you prefer to stay consistent with OpenUI5 - I will leave as it is now.

Reply:
After a short discussion we decided to change enum bvalues as proposed.